### PR TITLE
Update shopping-cart quickstart with WildFlyInitialContextFactory and wildfly-config.xml

### PR DESCRIPTION
**Pull request by sudeshnas93**
Issue: https://issues.redhat.com/browse/WFLY-16214

### CODE REVIEW

**Comment by emmartins** [comment]
@sudeshnas93 @iweiss please update local cloned main branch and rebase this, there are CI fixes and improvements that will only apply by doing that.

**Comment by iweiss** [comment]
@emmartins should be good now

File: shopping-cart/client/src/main/java/org/jboss/as/quickstarts/client/Client.java [hunk 1]
```diff
@@ -34,15 +35,15 @@ public class Client {
     private static final String ACCESSORIES_1 = "Wireless Ergonomic Keyboard and Mouse";
     private static final String ACCESSORIES_2 = "32 GB USB 2.0 Flash Drive";
 
-    public static void main(String[] args) throws NamingException {
+    public static void main(String[] args) throws NamingException, PrivilegedActionException, InterruptedException {
```
[the remark this answers is not in the document]
Comment (sudeshnas93) [reply shopping-cart/client/src/main/java/org/jboss/as/quickstarts/client/Client.java]: Made the changes. It should be good now.

**Comment by chengfang** [comment]
all checks failed due to checkstyle errors. @sudeshnas93  can you fix it?

**Comment by sudeshnas93** [comment]
> all checks failed due to checkstyle errors. @sudeshnas93 can you fix it?

fixed it. :)

**Comment by sudeshnas93** [comment]
@chengfang @emmartins can you please review this PR?
thanks.

**Comment by chengfang** [comment]
LGTM (already approved in May)

**Comment by chengfang** [comment]
@emmartins Can you check again and merge if everything is ok?

**Pull request merged**